http/okhttp/src/main/java/com/okta/commons/http/okhttp/OkHttpRequestExecutor.java

```diff
@@ -196,5 +196,10 @@ public void writeTo(BufferedSink sink) throws IOException {
                 Util.closeQuietly(source);
             }
         }
+
+        @Override
+        public long contentLength() {
+            return ((ByteArrayInputStream) inputStream).available();
```

Comment:
Avoid casting here, if this implementation requires a BAIS, then set the type to match:
```diff
- private final InputStream inputStream;
+ private final ByteArrayInputStream inputStream;
```

Reply:
Does this need a null check?

Reply:
Yeah, sorry for this.
Fixed.